packages/lib-classifier/src/store/RootStore.js

```diff
@@ -97,6 +98,10 @@ const RootStore = types
       onPatch(self, _onPatch)
     }
 
+    function setLocale (newLocale) {
```

Comment:
Consider adding a related basic test

Reply:
Thanks for the note! I added tests for `setLocale()` to the RootStore